jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -197,10 +197,6 @@
         job-name: ci-kubernetes-e2e-gce-serial-release-1.2
         frequency: 'H H/6 * * *' # 4 times a day for older jobs.
         trigger-job: 'kubernetes-build-1.2'
-    - kubernetes-e2e-gce-ingress-release-1.2:
```

Comment:
Will you also delete the corresponding test-infra/jobs/ci-kubernetes-e2e-gce-ingress-release-1.2.sh (etc) jobs?